janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/config/job/AbstractDistCacheConfigurer.java

```diff
@@ -146,7 +146,7 @@ private static Conf configureByClasspath(String mapredJarFilename) {
         for (String cpentry : classpath.split(File.pathSeparator)) {
             if (cpentry.toLowerCase().endsWith(".jar") || cpentry.toLowerCase().endsWith(".properties")) {
                 paths.add(new Path(cpentry));
-                if (cpentry.toLowerCase().endsWith(mrj));
+                if (cpentry.toLowerCase().endsWith(mrj))
```

Comment:
@mbrukman Good point. But we might be okay. It definitely looks like a bug but more importantly it doesn't look like the actual JAR selected by this routine is relevant. The only usage originates from [Hadoop2Compat](https://github.com/JanusGraph/janusgraph/blob/bfa3c970555932d4983155168322f72297aaee85/janusgraph-hadoop-parent/janusgraph-hadoop-2/src/main/java/org/janusgraph/hadoop/compat/h2/Hadoop2Compat.java#L78-L84) where there's a comment that the actual JAR name (which corresponds to `mapredJarFilename` or `mrj` here) doesn't matter. I think if it did we would be seeing issues in Hadoop jobs since the incorrect job JAR is being used.